templates/openstack/install.html

```diff
@@ -121,9 +121,10 @@ <h2>Single-node, multi-node or large-scale cluster?</h2>
     <h2>Single-node OpenStack deployment</h2>
     <p class="u-sv3">These instructions use <a href="https://snapcraft.io/microstack" class="p-link--external">MicroStack</a> &dash; OpenStack in a snap, MicroStack is a pure upstream OpenStack distribution designed for small scale and edge deployments, that can be installed and maintained with a minimal effort.</p>
     <div class="p-notification">
-      <p class="p-notification__response">
-        <span class="p-notification__status">NOTE:</span> MicroStack is in a beta state. We encourage you to test it, give us your <a class="p-link--external" href="https://bugs.launchpad.net/microstack">feedback</a> and <a class="p-link--external" href="https://discourse.juju.is/search?q=microstack">ask questions</a>.
-      </p>
+      <div class="p-notification__content">
+        <span class="p-notification__title">NOTE:</span>
```

Comment:
```suggestion
        <span class="p-notification__title">Note:</span>
```
Lower-case to match other instances of notes